htdocs/web_portal/GOCDB_monitor/index.php

```diff
@@ -1,12 +1,14 @@
-<?php
-
-echo("<br/><H1 align='center'>GOCDB MONITOR</H1><hr/>");
+<br>
```

Comment:
Do you not like marking void tags with a trailing slash?

Reply:
That convention comes from XHTML and is invalid in HTML5.

Reply:
It should probably be removed anyway, as it's technically an invalid use of a line break.

Reply:
It's not invalid in HTML5, [it's optional](https://www.w3.org/TR/html5/syntax.html#start-tags), but I'd be happy to agree that it's unnecessary.

Reply:
:man_shrugging: